config/routes.rb

```diff
@@ -1,6 +1,12 @@
 # frozen_string_literal: true
 
 Rails.application.routes.draw do
+  namespace :admin do
```

Comment:
In the future - absolutely. For now since we have no concept of an admin user (just the SNAP applicants) we have no way to tell whether someone is an admin.

As a stop-gap I've put some ENV variables that we can use for basic auth username and password. They've also been added to heroku's config on both staging AND production. So if/when you would like to fire up the admin in either of those two environments, do a `heroku config | grep ADMIN_` (or along those lines) to get that info :)